src/main/java/org/prebid/server/handler/CookieSyncHandler.java

```diff
@@ -62,7 +71,9 @@ public void handle(RoutingContext context) {
             cookieSyncRequest = Json.decodeValue(body, CookieSyncRequest.class);
         } catch (DecodeException e) {
             logger.info("Failed to parse /cookie_sync request body", e);
-            context.response().setStatusCode(400).setStatusMessage("JSON parse failed").end();
+            final int status = 400;
```

Comment:
Literal 400 can be replaced with `HttpResponseStatus.BAD_REQUEST.code()` as in case above to be in the same style.